Cargo.toml

```diff
@@ -23,17 +23,14 @@ debug_assert_with_mut_call = "warn"
 
 [workspace.dependencies]
 pyo3 = "0.19.0"
-hugr = "0.12.1"
-hugr-cli = "0.6.0"
-hugr-llvm = "0.5.0"
 serde_json = "1.0.111"
-inkwell = "0.4.0"
 cargo_toml = "0.20.4"
 thiserror = "1.0.37"
+hugr-llvm = "0.5.1"
 
 [patch.crates-io]
 
 # Uncomment these to test the latest dependency version during development
 #hugr = { git = "https://github.com/CQCL/hugr", rev = "6bf6c82c9ec9d801ab43e311e5f815a3eea7d9c1" }
 #hugr-cli = { git = "https://github.com/CQCL/hugr", rev = "6bf6c82c9ec9d801ab43e311e5f815a3eea7d9c1" }
-#hugr-llvm = { git = "https://github.com/CQCL/hugr-llvm", commit = "7245ca91b45b828ddb49456e9e0a895d79f1d739" }
+hugr-llvm = { git = "https://github.com/CQCL/hugr-llvm", branch = "doug/reexport-deps" }
```

Comment:
When will we be able to remove the patch?
Add an issue and link it as a TODO here, so we don't forget.